# This message contains the structure of which the vector in imageCircleData.msg is made of. 
# One place in the vector contains x, y, z and size values of one circle detected in the image.
# Date: 04-12-2018	
# By Giel Oomen

float64 x	# Circle position on x-axis
float64 y	# Circle position on y-axis
float64 z	# Circle position on z-axis (depth in meters)
float64 size	# Circle size 
